Makefile

```diff
@@ -68,7 +68,7 @@ CARGO_CMD_PREFIX += $(if $(filter n,$(MAKEFLAGS)),,+)
 # Force cargo not to print progress bar
 CARGO_CMD_PREFIX += CARGO_TERM_PROGRESS_WHEN=never CI=1
 # Set PQ_LIB_DIR to make sure `storage_controller` get linked with bundled libpq (through diesel)
-CARGO_CMD_PREFIX += PQ_LIB_DIR=$(POSTGRES_INSTALL_DIR)/v16/lib
+CARGO_CMD_PREFIX += PQ_LIB_DIR=$(POSTGRES_INSTALL_DIR)/v17/lib
```

Comment:
Is there any particular reason for it?

I'm not against it, we just have a bunch of other places that should be updated as well:
- https://github.com/neondatabase/neon/blob/b987648e713b7842be1a41ab2eca898a9c90eefb/.github/workflows/_build-and-test-locally.yml#L160
- https://github.com/neondatabase/neon/blob/b987648e713b7842be1a41ab2eca898a9c90eefb/.github/workflows/_build-and-test-locally.yml#L212
- https://github.com/neondatabase/neon/blob/94e4a0e2a0d43e066bd006a68eb147333ab0d074/.github/workflows/neon_extra_builds.yml#L139
- https://github.com/neondatabase/neon/blob/3d380acbd1dd878b908df860a541ce77bd4506f3/Dockerfile#L48
- https://github.com/neondatabase/neon/blob/0bd8eca9ca98ffbcaa403a47b2e6dfc2961fafa1/control_plane/src/storage_controller.rs#L56